LuisPalominoTrevilla/TaskManagerSystem
# Feat/habits/GetHabits

## Changes:
- Added Get endpoint logic such that you can either get all habits, or a specific one.

## How to test:
- In Postman (or any other similar software) create a new request using the GET method.
- There are two main test scenarios which can be tested: getting all habits at once, and getting one specific habit using the habit id.
- For the first test scenario, it's sufficient to only send a GET request to localhost:4001/api/habits/. You should get an array of habits (note that you should first create the habits using the CreateHabit endpoint so you can create some registries in your local database)
- For the second test scenario, you will again send a GET request to localhost:4001/api/habits/, but you shall now append a parameter called id, as such: localhost:4001/api/habits/?id={habitID}
- Images below shown for reference:
![image](https://user-images.githubusercontent.com/31036932/57592332-c3acb600-74fb-11e9-8aff-e47812d34c21.png)
![image](https://user-images.githubusercontent.com/31036932/57592352-dd4dfd80-74fb-11e9-87a5-a7d268e77806.png)
